docs/Analyzer Configuration.md

```diff
@@ -408,4 +408,15 @@ Option Values:
 
 Default Value: `Heuristic`.
 
-Example: `dotnet_code_quality.CA1712.enum_values_prefix_trigger = AnyEnumValue`
\ No newline at end of file
+Example: `dotnet_code_quality.CA1712.enum_values_prefix_trigger = AnyEnumValue`
+
+### Specify IFormatProvider
+Option Name: `exclude_tryparse_methods`
```

Comment:
All the other boolean options seem to be working on a negative way so I have mimic this behavior. If you prefer another mechanism, please let me know.